ee/tabby-db/src/cache.rs

```diff
@@ -35,3 +36,52 @@ impl<T> Cache<T> {
         }
     }
 }
+
+pub struct TimedKeyedCache<K, V> {
```

Comment:
```suggestion
pub struct KeydCache<K, V> {
```

Reply:
I did want to have some indication that this is a timed cache, since the regular `Cache` isn't